lib/Backends/CPU/libjit/libjit.cpp

```diff
@@ -883,6 +883,17 @@ void libjit_scatterassign_i8(int8_t *data, const size_t *indices,
   libjit_scatterassign(data, indices, slices, numIndices, sliceSize);
 }
 
+void libjit_lengths_to_ranges_u(size_t *ranges, const size_t *lengths,
```

Comment:
What's the naming magic here? `libjit_function_type`?

Reply:
"lenghths_to_ranges" is the name defined in LLVMIRGen.cpp(line 1676) , and then the function name in kernel is "libjit_" + name defined in LLVMIRGen.cpp(here is `lenghths_to_range`s)" + "_" + type of the output of this instr (here is `u`).